plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/IOpenShiftConnection.java

```diff
@@ -22,7 +24,7 @@
  * @author jeff.cantrill
  *
  */
-public interface IOpenShiftConnection {
+public interface IOpenShiftConnection extends IConnection {
```

Comment:
This interface intentionally did not extend IConnection because things like 'username' and 'password' are not necessarily relevant to v3 openshift connections; they rely specifically on a token.  Additionally, I thought there were advantages in some of our consumers that facilitated testing without having to provide an IConnection.  Maybe it is immaterial now.